### ISSUE TITLE
Where is the latest version published?

### ISSUE THREAD

**Issue by voddan** [corda/corda-gradle-plugins#276]
I need to use the latest version of the gradle plugins (4.0.47) because they have a bugfix that affects me, but I can't figure out which repository they are published to.

I tried:
buildscript {

  repositories {
    mavenLocal()
    mavenCentral()
    jcenter()
    maven { url "https://plugins.gradle.org/m2/" }
  }

  dependencies {
// option 1
    classpath "gradle.plugin.net.corda.plugins:cordformation:4.0.47"
// option 2
    classpath "net.corda.plugins:cordformation:4.0.47"
  }
}

The error I get:
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find net.corda.plugins:cordapp:4.0.47.
     Searched in the following locations:
       - file:/home/voddan/.m2/repository/net/corda/plugins/cordapp/4.0.47/cordapp-4.0.47.pom
       - file:/home/voddan/.m2/repository/net/corda/plugins/cordapp/4.0.47/cordapp-4.0.47.jar
       - https://repo.maven.apache.org/maven2/net/corda/plugins/cordapp/4.0.47/cordapp-4.0.47.pom
       - https://repo.maven.apache.org/maven2/net/corda/plugins/cordapp/4.0.47/cordapp-4.0.47.jar
       - https://jcenter.bintray.com/net/corda/plugins/cordapp/4.0.47/cordapp-4.0.47.pom
       - https://jcenter.bintray.com/net/corda/plugins/cordapp/4.0.47/cordapp-4.0.47.jar
       - https://plugins.gradle.org/m2/net/corda/plugins/cordapp/4.0.47/cordapp-4.0.47.pom
       - https://plugins.gradle.org/m2/net/corda/plugins/cordapp/4.0.47/cordapp-4.0.47.jar
     Required by:
         project : > gradle.plugin.net.corda.plugins:cordformation:4.0.47

**Comment by voddan** [comment]
Ok I found all releases at 
    maven { url 'https://ci-artifactory.corda.r3cev.com/artifactory/corda-releases' }
so my problem is solved.

The only place where this URL can be found AFAIK is in the source code for corda examples.
I think it would be appropriate to also have it in the README for this repo.

**Comment by chrisr3** [comment]
The Gradle plugins are all published to Gradle's own plugins repository these days, so you only need to do the following:
```
plugins {
    id 'net.corda.plugins.cordapp' version '4.0.47'
}
```

**Comment by chrisr3** [comment]
The `quasar-utils` and `cordformation` plugins have a runtime dependency on the `cordapp` plugin, which was fine when all plugins were added to the `buildscript` repositories. But this doesn't seem to work when you use the new `plugins {}` block exclusively :man_facepalming:.

New plugins 4.0.48 and 5.0.7 will be released shortly...

**Comment by chrisr3** [comment]
Fixed by versions 4.0.48 and 5.0.7.
